src/dotnet/App.Maui/App.Maui.csproj

```diff
@@ -127,6 +127,12 @@
     <None Include="**\**\*.Android.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
     <Compile Remove="**\Android\**\*.cs" />
     <None Include="**\Android\**\*.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
+    <Compile Update="Platforms\iOS\IOSNotificationPermissions.cs">
+      <ExcludeFromCurrentConfiguration>false</ExcludeFromCurrentConfiguration>
```

Comment:
what does this property exactly do? unfortunately google returns nothing

Reply:
I have removed them